src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs

```diff
@@ -78,14 +79,165 @@ public override string Name
         }
 
         /// <summary>
-        ///  
+        ///
+        /// </summary>
+        public class ConsoleColorProxy
```

Comment:
This should be `internal class ConsoleColorProxy` since the only intended way to access it is via the PSObject. The empty documentation comment can then be removed as well

Reply:
@SeeminglyScience Sorry I delete my comments in your review, afterwards I notice that the error I get with travis was from public members. I didn't notice that it work with internal members. And when I push that change AppVeyor worked I don't know why it was stuck for 20 minute.

Reply:
AppVeyor and Travis sometimes have issues (not our issues) and it's just a matter of restarting the build and it works. 😀